snapper.py

```diff
@@ -535,7 +563,8 @@ def get_snapraid_config():
     with open(config_file, 'r') as file:
         snapraid_config = file.read()
 
-    file_regex = re.compile(r'^(content|(?:\d+-)?parity) +(.+/\w+.(?:content|(?:\d+-)?parity)) *$',
+    #Split parity handling
+    file_regex = re.compile(r'^(content|parity) +(.+/\w+.(?:content|parity)) *$',
```

Comment:
Also this

Reply:
Same as above - I'll amend this.  Split parity is obviously already handled in your regex.